devtools/db_import/db_import/rules.py

```diff
@@ -71,9 +71,9 @@ def read_file(filepath):
       contents = fd.read()
 
     if dump_files_to:
-      dump_directory = dump_files_to / config["bucket_name"]
-      dump_directory.mkdir(parents=True, exist_ok=True)
-      (dump_directory / filepath).write_text(contents)
+      full_filepath = dump_files_to / config["bucket_name"] / filepath
+      full_filepath.parent.mkdir(parents=True, exist_ok=True)
+      full_filepath.write_text(contents)
```

Comment:
This is a bugfix that I discovered while testing the download tool. The subdirectories were not created correctly. I also adjusted the unit tests so that would have been caught.